Sazabi.cpp

```diff
@@ -1585,6 +1619,24 @@ void CSazabi::UnInitializeObjects()
 }
 int CSazabi::ExitInstance()
 {
+	PROC_TIME(ExitInstance)
+	// 以下の理由からここでは排他制御をおこなう
+	// * CloseAllで複数のプロセスでこの部分を通る可能性がある
+	// * InitInstanceでのInitMultipleInstanceと本メソッドのIsExistsAnotherInstanceが
+	//   並列実行されると、他のインスタンスの存在が正しく判定されない可能性がある。
+	HANDLE hMutex = {0};
+	DWORD dwWaitResult = WAIT_FAILED;
+	CHRONOS_ENTER_CRITICAL_SECTION(hMutex, dwWaitResult, 30 * 1000);
+	if (!hMutex)
+	{
+		WriteDebugTraceDateTime(_T("Failed to CreateMutex"), DEBUG_LOG_TYPE_GE);
+	}
+	if (hMutex && dwWaitResult != WAIT_OBJECT_0)
+	{
+		CString logmsg;
+		logmsg.Format(_T("Failed to aquire mutex: %lx"), dwWaitResult);

```

Comment:
It might be better to be identified by each message.
same message is use in CSazabi::ExitInstance and CSazabi::InitInstance.
It seems that it is not appropriate.

Reply:
Thanks, I have added the method names to the logs.